packages/pharos/src/components/popover/pharos-popover.ts

```diff
@@ -38,11 +38,11 @@ export class PharosPopover extends ScopedRegistryMixin(FocusMixin(OverlayElement
   public isOnBackground = false;
 
   /**
-   * Indicates the aria label to apply to the dialog.
-   * @attr label
+   * Indicates the aria label to apply to the button.
```

Comment:
I'm not sure PharosPopover ever _renders_ a button. Looks like the label goes on a `div` in this case.